### PR TITLE
[142] Develop a Contact Page using HTML

### PR DESCRIPTION
**Pull request by RafiatAmin**
Screen Shot 2020-12-03 at 10 36 33 AM

### CODE REVIEW

File: submissions/rafiatAminu/html-forms/contact.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            <input type="text" name="firstname" id="firstname" /><br>
+            <label for="lastname">Last Name</label><br>
+            <input type="text" name="lastname" id="lastname" /><br>
+            <label for="email">Enter your email address</label><br>
+            <input type="text" name="email" id="email" /><br>
+            <label for="subject">Subject</label><br>
+            <input type="text" name="firstname" id="firstname" /><br>
+            <label for="message">Message Input</label><br>
+            <textarea name="message" id="message" cols="50" rows="10"></textarea><br>
+
```
Comment (Ireti203) [inline submissions/rafiatAminu/html-forms/contact.html:34]: Remove whitespace

File: submissions/rafiatAminu/HTML/html-forms/contact.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <input type="text" name="firstname" id="firstname"><br>
+            <label for="lastname">Last Name</label><br>
+            <input type="text" name="lastname" id="lastname"><br>
+            <label for="email">Enter your email address</label><br>
+            <input type="text" name="email" id="email"><br>
+            <label for="subject">Subject</label><br>
+            <input type="text" name="firstname" id="firstname"><br>
+            <label for="message">Message Input</label><br>
+            <textarea name="message" id="message" cols="50" rows="10"></textarea><br>
+
```
Comment (Ireti203) [inline submissions/rafiatAminu/HTML/html-forms/contact.html:34]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
You have two contact.html files. Kindly remove the one inside `HTML` folder

File: submissions/rafiatAminu/html-forms/contact.html (hunk 1 above)
Comment (RafiatAmin) [reply submissions/rafiatAminu/html-forms/contact.html:34]: Screen Shot 2020-12-08 at 9 14 26 AM

Here's a screenshot of my code, there's no whitespace. I have formatted countless times

**Pull request closed without merging**